vespalib/src/tests/stllike/hashtable_test.cpp

```diff
@@ -134,6 +134,27 @@ TEST("require that hashtable<vector<int>> can be copied") {
     EXPECT_EQUAL(6, (*table.find(2))[2]);
 }
 
+/**
+ * Test to profile destruction and recreation of hash map.
+ * It revealed some unexpected behaviour. Results with 10k iterations on 2018 macbook pro 2.6 Ghz i7
+ * 1 - previous - 14.7s hash_node() : _node(), _next(invalid) {}
+ * 2 - test     -  6.6s hash_node() : _next(invalid) { memset(_node, 0, sizeof(node)); }
+ * 3 - current  -  2.3s hash_node() : _next(invalid) {}
+ */
+TEST("benchmark clear") {
+    vespalib::hash_map<uint32_t, uint32_t> m(1000000);
```

Comment:
Would it be useful to also benchmark non-POD types for clearing?

Reply:
Will do.

Reply:
Fixed in #18125